apps/web/modules/event-types/views/event-types-listing-view.tsx

```diff
@@ -978,28 +946,14 @@ export const EventTypesCTA = ({ initialData, filters }: Props) => {
         };
       }) ?? [];
 
-  return <CTA profileOptions={profileOptions} isOrganization={!!user?.organizationId} />;
+  return <CTA profileOptions={profileOptions} />;
 };
 
-const EventTypesPage = ({ initialData, filters }: Props) => {
-  const { data: user } = useMeQuery();
-  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+const EventTypesPage = ({ getUserEventGroupsData, user }: Props) => {
```

Comment:
NIT: while we're here, can we rename `getUserEventGroupsData` to `userEventGroupsData`? It looks like a function, while it's an object.

Reply:
agreeed